slide-in-with-cta/template.css

```
@@ -0,0 +1,204 @@
+    height: 100%;
+    min-height: 1px;
+}
+
+#evg-slide-in-with-cta .evg-header,
+#evg-slide-in-with-cta .evg-subheader,
+#evg-slide-in-with-cta .evg-cta {
+    display: block;
+    max-width: 90%;
+    height: fit-content;
+    text-align: center;
+}
+
+/* ==========================================================================
+   Headers
+========================================================================== */
+
+#evg-slide-in-with-cta .evg-header,
+#evg-slide-in-with-cta .evg-subheader {
+    margin-bottom: 0.5rem;
+}
+
+#evg-slide-in-with-cta .evg-header {
+    font-size: 1.75rem;
+    line-height: 1.75rem;
+}
+
+#evg-slide-in-with-cta .evg-subheader {
+    font-size: 1.5rem;
+    font-weight: 300;
+    line-height: 1.5rem;
+}
+
+/* ==========================================================================
+   Call-To-Action Button
+========================================================================== */
+
+#evg-slide-in-with-cta .evg-btn {
+    vertical-align: middle;
+    margin-top: 1rem;
+    padding: 0.5rem 1rem;
+    border: 1px solid transparent;
+    border-radius: 3rem;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.5;
+    text-align: center;
+    text-decoration: none;
+    transition:
+        color 0.15s ease-in-out,
+        background-color 0.15s ease-in-out,
+        border-color 0.15s ease-in-out,
+        box-shadow 0.15s ease-in-out;
+    cursor: pointer;
+    -webkit-user-select: none;
+    -moz-user-select: none;
+    -ms-user-select: none;
+    user-select: none;
+}
+
+#evg-slide-in-with-cta .evg-btn-primary {
+    border-color: #fff;
+    color: #000;
+    background-color: #fff;
+    transition: background-color 0.15s, color 0.15s;
+}
+
+#evg-slide-in-with-cta .evg-cta:hover {
+    text-decoration: none;
+}
+
+#evg-slide-in-with-cta .evg-btn-dismissal {
+    position: absolute;
+    top: 0;
+    right: 0;
+    padding: 0 0.5rem;
+    border: 1px solid transparent;
+    color: #fff;
+    background-color: transparent;
+    font-size: 2rem;
+    font-weight: 700;
+    line-height: 1;
+    cursor: pointer;
+}
+
+/* ==========================================================================
+   Visibility
+========================================================================== */
+
+#evg-slide-in-with-cta .evg-hide {
+    visibility: hidden;
+    display: none;
+    width: 0;
+    height: 0;
+}
+
+/* ==========================================================================
+   Style
+========================================================================== */
+
+#evg-slide-in-with-cta.evg-dark-on-light {
+    background-color: #fff;
+}
+
+#evg-slide-in-with-cta.evg-light-on-dark {
+    background-color: #000;
+}
+
+#evg-slide-in-with-cta .evg-header.evg-dark-on-light,
+#evg-slide-in-with-cta .evg-subheader.evg-dark-on-light,
+#evg-slide-in-with-cta .evg-btn-dismissal.evg-dark-on-light,
+#evg-slide-in-with-cta .evg-cta.evg-light-on-dark {
+    color: #000;
+    background-color: #fff;
+}
```

This can be simplified to this
```suggestion
#evg-slide-in-with-cta .evg-dark-on-light {
    color: #000;
    background-color: #fff;
}
```